### PR TITLE
how-to/deploy: update a file name for TiDB Binlog

### PR DESCRIPTION
**Pull request by yilongrong**
当前v3.0.2版本中这里已经修改为pump-cluster.yml文件名称.

### CODE REVIEW

File: v3.0/how-to/deploy/tidb-binlog.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Pump 和 Drainer 均可部署和运行在 Intel x86-64 架构的 64 位通用硬
         172.16.10.74
         ```
 
-        默认 Pump 保留 7 天数据，如需修改可修改 `tidb-ansible/conf/pump.yml` 文件中 `gc` 变量值，并取消注释。
+        默认 Pump 保留 7 天数据，如需修改可修改 `tidb-ansible/conf/pump-cluster.yml` (旧版本为pump.yml)文件中 `gc` 变量值，并取消注释。
```
Comment (lilin90) [inline v3.0/how-to/deploy/tidb-binlog.md]:
```suggestion
        默认 Pump 保留 7 天数据，如需修改可修改 `tidb-ansible/conf/pump-cluster.yml`（TiDB 3.0.1 及之前版本中为 `pump.yml`）文件中 `gc` 变量值，并取消注释。
```

Comment (lilin90) [reply v3.0/how-to/deploy/tidb-binlog.md]: Please also help update this file in `dev`: `dev/how-to/deploy/tidb-binlog.md`.

**Comment by july2993** [comment]
@liubo0127 PTAL

File: v3.0/how-to/deploy/tidb-binlog.md (hunk 1 above)
Comment (liubo0127) [reply v3.0/how-to/deploy/tidb-binlog.md]: In v3.0.0 v3.0.1 v3.0.2 version is `pump-cluster.yml`, after v3.0.3 version changed to `pump.yml`.

Comment (lilin90) [reply v3.0/how-to/deploy/tidb-binlog.md]: @liubo0127 Got it! 
@yilongrong I updated my suggestion based on liubo's comment. Please address the comment and also apply changes to `dev/how-to/deploy/tidb-binlog.md`.

**Comment by yikeke** [comment]
@liubo0127 PTAL, this PR is old.

**Comment by yikeke** [comment]
Both `dev` and `v3.0` have been updated. PTAL again. @liubo0127 @lilin90

File: dev/reference/tools/tidb-binlog/deploy.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ Pump 和 Drainer 均可部署和运行在 Intel x86-64 架构的 64 位通用硬
         172.16.10.74
         ```
 
-        默认 Pump 保留 7 天数据，如需修改可修改 `tidb-ansible/conf/pump.yml` 文件中 `gc` 变量值，并取消注释。
+        默认 Pump 保留 7 天数据，如需修改可修改 `tidb-ansible/conf/pump.yml`（TiDB 3.0.0~3.0.2 版本中为 `pump-cluster.yml`）文件中 `gc` 变量值，并取消注释。
```
Comment (yikeke) [inline dev/reference/tools/tidb-binlog/deploy.md]:
```suggestion
        默认 Pump 保留 7 天数据，如需修改可修改 `tidb-ansible/conf/pump.yml`（TiDB 3.0.0~3.0.2 版本中为 `tidb-ansible/conf/pump-cluster.yml`）文件中 `gc` 变量值，并取消注释。
```

File: v3.0/reference/tools/tidb-binlog/deploy.md [hunk 3]
```diff
@@ -56,7 +56,7 @@ Pump 和 Drainer 均可部署和运行在 Intel x86-64 架构的 64 位通用硬
         172.16.10.74
         ```
 
-        默认 Pump 保留 7 天数据，如需修改可修改 `tidb-ansible/conf/pump.yml` 文件中 `gc` 变量值，并取消注释。
+        默认 Pump 保留 7 天数据，如需修改可修改 `tidb-ansible/conf/pump.yml`（TiDB 3.0.2 及之前版本中为 `pump-cluster.yml`）文件中 `gc` 变量值，并取消注释。
```
Comment (yikeke) [inline v3.0/reference/tools/tidb-binlog/deploy.md]:
```suggestion
        默认 Pump 保留 7 天数据，如需修改可修改 `tidb-ansible/conf/pump.yml`（TiDB 3.0.2 及之前版本中为 `tidb-ansible/conf/pump-cluster.yml`）文件中 `gc` 变量值，并取消注释。
```

**Pull request merged**